cmd/vsphere-csi/main.go

```diff
@@ -58,20 +59,31 @@ func main() {
 	if err != nil {
 		log.Errorf("Failed retrieving cluster flavor. Error: %v", err)
 	}
-	serviceMode := os.Getenv("X_CSI_MODE")
+	serviceMode := os.Getenv(csitypes.EnvVarMode)
 	commonco.SetInitParams(ctx, clusterFlavor, &service.COInitParams, *supervisorFSSName, *supervisorFSSNamespace,
 		*internalFSSName, *internalFSSNamespace, serviceMode)
 
-	gocsi.Run(
-		context.Background(),
-		csitypes.Name,
-		"A CSI plugin for vSphere Cloud Native Storage",
-		usage,
-		provider.New())
-}
-
-const usage = `    VSPHERE_CSI_CONFIG
+	if *useGocsi {
+		const usage = `VSPHERE_CSI_CONFIG
         Specifies the path to the csi-vsphere.conf file
-
         The default value is "/etc/cloud/csi-vsphere.conf"
```

Comment:
Can we print a log to help debug if we are running driver with gocsi or non-gocsi.